package.json

```diff
@@ -33,7 +33,7 @@
     "mobile-detect": "^1.4.1",
     "moving-average": "^1.0.0",
     "naf-janus-adapter": "https://github.com/mozilla/naf-janus-adapter#feature/disconnect",
-    "networked-aframe": "https://github.com/mozillareality/networked-aframe#mr-social-client/master",
+    "networked-aframe": "github:mozillareality/networked-aframe#feature/sound-source-set",
```

Comment:
We have a few PRs in flight again that depend on unlanded NAF changes, we should use the `mr-social-client/master` integration branch again

Reply:
:+1: will do